lib/policyEvaluator/RequestContext.js

```diff
@@ -35,27 +35,44 @@ function _findAction(service, method) {
     if (service === 's3') {
         return _actionMap[method];
     }
+    if (service === 'utapi') {
```

Comment:
This will be expanding at some point so I want to be clear which service is being used.  When we get beyond 3 items, we can organize with an object or map structure.